src/libmongoc/tests/TestSuite.c

```diff
@@ -118,6 +118,7 @@ _handle_signal (int signum)
 {
    const char *s = "\nProcess was interrupted by the delivery of a signal.\n";
    const char *sigstr;
+   size_t n;
```

Comment:
Why the addition of this write-only variable? It seems as if it is to address nodiscard warnings on calls to `write()`, but if that is the case, a cast to `(void)` should suffice.

Reply:
I thought so too. It seems like GCC (8.x) doesn't consider the `(void)write(...)` to be enough 😕.